src/tests/java17/StrongEncapsulationTest.java

```diff
@@ -0,0 +1,22 @@
+package java17;
+
+import gov.nasa.jpf.util.test.TestJPF;
+import org.junit.Test;
+import java.lang.reflect.Method;
+
+public class StrongEncapsulationTest extends TestJPF {
+
+    @Test
+    public void testAccessPrivateMethods() {
+        try {
+            Method method = String.class.getDeclaredMethod("valueOf", Object.class);
+            method.setAccessible(true);
```

Comment:
Method `valueOf` is public by default, so calling `setAccessible` does not change anything.
A `SecurityManager` setting may deny the usage of `setAccessible`, but we do not support that feature in JPF at the moment.